README.md

```diff
@@ -87,6 +88,36 @@ Having issues? See [FAQs](./docs/faqs.md).
 
 See [prelease.md](docs/prerelease.md)
 
+### Implementation examples
+
+[We decided](https://github.com/trussworks/react-uswds/issues/2001) to focus on our [storybook](https://trussworks.github.io/react-uswds/) over hosting an example/boilerplate implementation, but you can find a few such things from our friends:
+
+* [NASA's Americana Remix stack](https://github.com/nasa-gcn/americana-stack?tab=readme-ov-file)
+* [USDS's Gatsby starter template](https://github.com/usds/gatsby-uswds-ts-starter)
+* [Nava's NextJS starter template](https://github.com/navapbc/template-application-nextjs)
+
+You can also check out some open source projects using React-USWDS across federal and state governments:
+
+* DOL
+  * [Unemployment Insurance Claimant Experience Pilot](https://github.com/USDepartmentofLabor/ui-claimant-experience-pilot)
+  * [New Jersey's Unemployment Insurance Claimant Intake](https://github.com/USDepartmentofLabor/UI-Claimant-Experience-NJ-Pilot)
```

Comment:
Here I'd maybe also use the word Pilot here, since their actual app is in a different repo and has evolved a good bit since this snapshot

```suggestion
  * [New Jersey's Unemployment Insurance Claimant Intake Pilot](https://github.com/USDepartmentofLabor/UI-Claimant-Experience-NJ-Pilot)
```